docs/Queries.md

```diff
@@ -63,6 +66,15 @@ GET requests) or the request body (for POST requests).
 An array of feature flag (Runtime Flag) names that should be returned
 alongside the main request.
 
+### `meta`
+
+#### `method`
+
+You can force the query to be sent with a particular HTTP method by specifying
+it here as a _lowercase_ string: `get`, `post`, `delete`, or `patch`. Note that
```

Comment:
It's annoying that the casing on HTTP methods is so inconsistent in different environments, but I guess you're right that that ship has sailed and I shouldn't try to enforce one or the other here.